spec/system/work_create_spec.rb

```diff
@@ -78,96 +81,250 @@
+          visit edit_work_path(work, params: { wizard: true })
 
-      work = Work.last
-      visit(work_path(work))
-
-      has_been_created_message = "#{title} has been created"
-      within("ul.work-messages") do
-        expect(page).to have_content(has_been_created_message)
-        expect(page).to have_content("#{title} is ready for review")
-        expect(page).to have_content(work.group.title)
-      end
-
-      click_on "Hide Messages"
-      expect(page).not_to have_content(has_been_created_message)
-      click_on "Show Messages"
-      expect(page).to have_content(has_been_created_message)
-
-      # Now sign is as the group moderator and see the notification on your dashboard
-      sign_out user
-      sign_in curator
-      visit(user_path(curator))
-      expect(page).to have_content curator.given_name
-      # This is the blue badge on the work that should show up for a curator
-      #  when a work is startend and marked completed by a submitter
-      within("#unfinished_datasets span.badge.rounded-pill.bg-primary") do
-        expect(page).to have_content "2"
+          expect(work.resource.related_objects.count).to eq(0)
+
+          click_on "Additional Metadata"
+          wait_for_ajax
+          find("tr:last-child input[name='related_objects[][related_identifier]']").set "https://related.example.com"
+          wait_for_ajax
+          click_on "Save Work"
+          work.reload
+          expect(work.resource.related_objects.count).to eq(0)
+          # rubocop:disable Layout/LineLength
+          expect(page).to have_content("1 error prohibited this dataset from being saved:\nRelated Objects are invalid: Related Identifier Type is missing or invalid for https://related.example.com, Relationship Type is missing or invalid for https://related.example.com")
+          # rubocop:enable Layout/LineLength
+        end
+      end
+    end
+
+    context "when there is no README attached to the Work" do
+      let(:work) do
+        FactoryBot.create(:draft_work, created_by_user_id: user.id)
+      end
+      it "renders the form for uploading a README", js: true do
+        sign_in user
+        visit work_readme_select_path(work, params: { wizard: true })
+
+        expect(page).to have_content("Please upload the README")
+        expect(page).to have_button("Continue", disabled: true)
+
+        path = Rails.root.join("spec", "fixtures", "files", "readme.txt")
+        attach_file(path) do
+          page.find("#patch_readme_file").click
+        end
+
+        click_on "Continue"
+        expect(page).to have_content("New Submission")
+
+        work.reload
+        stub_s3 data: [FactoryBot.build(:s3_readme, work:)]
+
+        readme = Readme.new(work, user)
+        expect(readme.file_name).not_to be nil
+        expect(readme.file_name).to eq("README.txt")
+      end
+
+      context "when a README has already been attached to a Work" do
+        xit "attempting to upload another README renders an error" do
```

Comment:
Is there a reason for adding a pending test?  We rarely change the pending tests in the future.